packages/office-ui-fabric-react/src/components/Dropdown/Dropdown.tsx

```diff
@@ -272,11 +261,69 @@ export class Dropdown extends BaseComponent<IDropdownInternalProps, IDropdownSta
     }
   }
 
+  private _calculateMaxHeight() {
```

Comment:
this logic looks expensive.. both to maintain and to execute.. finding all the nodes and explicitly measuring them..  again, `List` provides a lot of that (caching, estimation), why duplicate it here?